proto/query.proto

```diff
@@ -398,6 +399,9 @@ message QueryResult {
   repeated Row rows = 4;
   string info = 6;
   string session_state_changes = 7;
+  vttime.Duration execute_duration = 8;
+  vttime.Duration plan_duration = 9;
+  vttime.Duration commit_duration = 10;
```

Comment:
Would it make sense to wrap this in a `QueryStats` message instead?

So something like: 

```proto
message QueryStats {
  vttime.Duration execute = 1;
  vttime.Duration plan = 2;
  vttime.Duration commit = 3;
```

Then just add:

```
message QueryResult {
 ...
  QueryStats stats = x;
}```

Seems a bit better to wrap this in it's own Message imo, but I don't have strong opinions about this.

Reply:
Okay, this has been addressed.